internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+type PullRequestEvent struct {
+	Action      string      `json:"action"`
+	PullRequest PullRequest `json:"pull_request"`
+	Repository  Repository  `json:"repository"`
+}
+
+type GitHubUser struct {
+	Login string `json:"login"`
+	Id    int    `json:"id"`
```

Comment:
You may want to use more descriptive field names for `Id` in `GitHubUser`, `PullRequest`, and `Repository`. It's uncommon to use `Id` without a prefix to indicate which entity it belongs to (`GitHubUserId`, `PullRequestId`, etc.), which could lead to confusion during maintenance or code readability and also avoid ambiguity in larger codebases.